charts/vault/templates/post-pod-creation-cm.yaml

```diff
@@ -0,0 +1,34 @@
+{{- $major := .Capabilities.KubeVersion.Major -}}
+{{- $minor := .Capabilities.KubeVersion.Minor -}}
```

Comment:
Use https://helm.sh/docs/chart_template_guide/function_list/#semantic-version-functions instead of custom logic.

Reply:
Fixed thanks